### ISSUE TITLE
SmartThings and Groovy deprecation 

### ISSUE THREAD

**Issue by soupman98** [maartenvantjonger/omnilogic-smartapp#14]
Hi Maaren, What's the prognosis for this solution given the changes to Smart things?
Any plans to look at HomeAssistant?

**Comment by maartenvantjonger** [comment]
Hey Steve, Prognosis looking bleak. I personally only use Hubitat which will keep working with Groovy drivers.
I don't know Edge/ Lua and don't really have time and motivation to learn it and convert the code.

HomeAssistant seems to already have an Omnilogic implementation. Have you checked that out?
https://www.home-assistant.io/integrations/omnilogic/

**Comment by soupman98** [comment]
Hi Marrten.. Yeah. I'm pretty pissed at ST. I've started looking at HomeAssistant. but they're lacking the ActionTiles tie in.. Thanks for the update

**Comment by soupman98** [comment]
Looks like it died today. ...

**Comment by maartenvantjonger** [comment]
Sorry!

![Rip](https://user-images.githubusercontent.com/7673954/211960139-0dd41b46-d385-4ae7-9b54-b5c5c8e7e034.jpg)

**Comment by soupman98** [comment]
Yup it's me again.. I'm now running Hubitat (barely), HomeAssistant and Smartthings... primarily Smartthings still post Edge enablement, but loving the HomeAssisant automations. 

In your hubitat integration - is it full fidelity controls and visualization? e.g. on/off status and control for heater, pump, chlorinator, lights, relay (boht the 4 high voltage and low voltage?)
